### PR TITLE
removed the vs extension  from the build

### LINKED ISSUE #49: MAJORFIX: DOTNET BUILD issue
In VS the project builds and all tests pass

`========== Build: 7 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========`

image

BUT the DOTNET BUILD command fails which is what we use in the GitHub Actions.  (This can be reproduced locally running DOTNET BUILD in the Package Manager Console)

image

### PR DESCRIPTION
**Pull request by coeur85**
closes #49

### CODE REVIEW

**Comment by cjdutoit** [comment]
@coeur85 can you build and run the Build project.   The yaml file is not present, so we can't verify if this solved the issue.

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ name: .Net
 on:
   push:
     branches:
-    - main
+    - users/coeur85/FIx-buildFromCli
```
Comment (cjdutoit) [inline .github/workflows/dotnet.yml:5]: The branch name need to be main as this is our default branch

File: .github/workflows/dotnet.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ name: .Net
...
   pull_request:
     branches:
-    - main
+    - users/coeur85/FIx-buildFromCli
```
Comment (cjdutoit) [inline .github/workflows/dotnet.yml:8]: The branch name need to be main as this is our default branch

Comment (coeur85) [reply .github/workflows/dotnet.yml]: chnaged it back

**Pull request merged**